src/common/selectors.js

```diff
@@ -448,6 +448,15 @@ export const createGetTags = collectionSelectors => {
   return _extendCollectionSelector(getTags, 'tag')
 }
 
+export const createGetLastShutdownTime = () => create(
```

Comment:
Selectors defined in this module always:
- have `state` and `props` as inputs, or
- have their input selectors configurable